examples/cluster/main.tf

```diff
@@ -1,35 +1,32 @@
-terraform {
```

Comment:
create something like this https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/versions.tf